Specs/Number/Arabic/NumberModel.json

```diff
@@ -1417,19 +1407,19 @@
     ]
   },
   {
-    "Input": "تسعين وخمس مائة أخماس من الوزن",
+    "Input": "كم هي خمسة وتسعون مائة على خمسة ؟",
```

Comment:
Can you provide the reasons for the spec edits and removals in the PR description please?
Keep in mind that we want to keep as many variations as possible. Spec cases should only be removed if they really don't apply in a target language. The original writer of these specs was a native Arabic speaker from Egypt, so if these are not MSA, they may still be of frequent use.

Reply:
Sure @tellarin , I just added in PR Description.
FYI,  we identified some discrepancy in some cases and got re-corrected by Language Expert. 
Thanks